packages/lesswrong/server/recommendations.js

```diff
@@ -53,7 +53,7 @@ const recommendablePostFilter = {
   // Only consider recommending posts if they have score>30. This has a big
   // effect on the size of the recommendable-post set, which needs to not be
   // too big for performance reasons.
-  baseScore: {$gt: 30},
+  baseScore: {$gt: 60},
```

Comment:
30 was probably too low, but 60 seems probably too high. Maybe 45? In any case, the comment above this should be updated to match.

Reply:
60 was intended to make sense after running the script that actually will update the old posts, and the PR was intended in part to force us to actually do that.

Reply:
(agree with changing the comments, although I'm generally not a fan of code comments that are overly specific since they easily get out of date in cases like this)

Reply:
I don't think we should be forced to actually update the old votes. It might end up being a rabbit-hole, and I don't want to delay recommendations much longer.

Reply:
I could see this going as low as 50, but I find it fairly weird to see "recommendations" of posts with karma-totals that are, in the current zeitgeist, very low. I think there's also enough posts at higher karma levels for "we don't get around to re-running the votes" to be that big an issue if we set it to 50.